### PR TITLE
COR-1487-remove-toggle-on-positive-tested-page

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
## Summary

* 
* removal of the safety region choropleth on the national positive tested page
* 

#### Before
<details>
<summary>Toggle before screenshots</summary>

Screenshot 2023-03-28 at 14 21 44

</details>

#### After
<details>
<summary>Toggle after screenshots</summary>

Screenshot 2023-03-28 at 14 21 53

</details>

### CODE REVIEW

**Review by VWSCoronaDashboard26 — DISMISSED** [review]

**Review by VWSCoronaDashboard21 — COMMENTED** [review]
I think there needs to be some paddingTop on the Box around the legendaComponent.

**Comment by VWSCoronaDashboard18** [comment]
> I think there needs to be some paddingTop on the Box around the legendaComponent.

Nice catch! I have added the spacing to the (extra)element

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx [hunk 1]
```diff
@@ -345,70 +344,37 @@ function PositivelyTestedPeople(props: StaticProps<typeof getStaticProps>) {
               description={
                 <>
                   <Markdown content={textNl.map_toelichting} />
-                  {replaceComponentsInText(textNl.map_last_value_text, {
-                    infected_per_100k: <BoldText>{`${formatNumber(dataOverallLastValue.infected_per_100k)}`}</BoldText>,
-                    dateTo: formatDateFromSeconds(dataOverallLastValue.date_unix, 'weekday-long'),
-                  })}
+                  <Box marginBottom={space[3]}>
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:347]: Whats the need for now wrapping this in a Box? 🤔

**Review by VWSCoronaDashboard28 — APPROVED** [review]
I'm approving this, I have one small question/comment.

**Review by VWSCoronaDashboard21 — APPROVED** [review]
Nice job!

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx (hunk 1 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:347]: See the last comment. There is a need for spacing below this element.

**Pull request merged**